packages/web/config/ibc-assets.ts

```diff
@@ -784,6 +784,13 @@ export const IBCAssetInfos: (IBCAsset & {
     //   "https://blue.kujira.app/ibc?destination=kaiyo-1&source=osmosis-1&denom=ukuji",
     isVerified: true,
   },
+  {
```

Comment:
please move to the bottom of the list

Reply:
Done https://github.com/osmosis-labs/osmosis-frontend/pull/1085/commits/3fe56229141767847e9cf12660ae72714dea7d1f